tests/cypress/plugins/index.ts

```diff
@@ -48,6 +48,7 @@ module.exports = (on: Cypress.PluginEvents, config: Cypress.PluginConfigOptions)
   config.env.rsa_private_key_qa  = process.env.RSA_PRIVATE_KEY_QA;
   config.env.rsa_public_key_qa   = process.env.RSA_PUBLIC_KEY_QA;
   config.env.upgrade             = process.env.UPGRADE;
+  config.env.fleet_app_version   = process.env.FLEET_APP_VERSION;
```

Comment:
Suggestion: maybe this could be more exact with `FLEET_APP_VERSION_BEFORE_UPGRADE`

Reply:
No, I am getting the env variable as it is from the `master-e2e.yaml` file to Cypress.
The string processing i.e. splitting and etc thing happened in the test so I use the `const` name as it is there.